localisation/00_zuwc_united_localisation.csv

```diff
@@ -2227,7 +2227,7 @@ has_unpleasant_trait_trigger_tooltip;Has unpleasant trait (deceitful, shy, paran
 is_my_educator_tooltip;Is my educator\n;;;;;;;;;;;;;x
 has_not_3_students_tooltip;Has not 3 students\n;;;;;;;;;;;;;x
 war_fires_by_event_tooltip;Is war target of event\n;;;;;;;;;;;;;x
-centaur_group_opinion;Centaur Rivalry Opinion;;;;;;;;;;;;;x
+centaur_group_opinion;Centaur Rivalry Opinion ;;;;;;;;;;;;;x
```

Comment:
Is there any reason why you added spaces to a lot of localization lines?

Reply:
No, they appeared after the merge and I don't really know why. Better check this merge again.
https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/pull/665/commits/b931ab83721c7e5547626b19c8484d4299768af6

Reply:
Fixed!